app/models/base_organisation.rb

```diff
@@ -3,9 +3,9 @@ class BaseOrganisation < ActiveRecord::Base
   friendly_id :slug_candidates, use: :slugged
   
   acts_as_paranoid
-  validates_url :website, prefferred_scheme: 'http://', message: 'Website is not a valid URL',
+  validates_url :website, preferred_scheme: 'http://', message: 'Website is not a valid URL',
     if: proc{|org| org.website.present?}
-  validates_url :donation_info, prefferred_scheme: 'http://', message: 'Donation url is not a valid URL',
+  validates_url :donation_info, preferred_scheme: 'http://', message: 'Donation url is not a valid URL',
```

Comment:
could we just break this over two lines - putting the message section on the next line?